<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamerMenu</class>
 <widget class="QWidget" name="GamerMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Library</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      font-size: 24px;
      font-weight: bold;
      color: #2C3E50;
     </string>
    </property>
    <property name="text">
     <string>Game Library</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="gameNameInput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>49</y>
      <width>500</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
                    border-radius: 10px;
                    padding: 10px;
                    }</string>
    </property>
    <property name="placeholderText">
     <string>Enter game name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #3498DB;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #2980B9;
      }
     </string>
    </property>
    <property name="text">
     <string>Buy Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findGameButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #1ABC9C;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #16A085;
      }
     </string>
    </property>
    <property name="text">
     <string>Find Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #F39C12;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #E67E22;
      }
     </string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #E74C3C;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #C0392B;
      }
     </string>
    </property>
    <property name="text">
     <string>Delete Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readLibraryButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #8E44AD;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #732D91;
      }
     </string>
    </property>
    <property name="text">
     <string>Read Library</string>
    </property>
   </widget>
   <widget class="QPushButton" name="devProfileButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #1ABC9C;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #16A085;
      }
     </string>
    </property>
    <property name="text">
     <string>You're Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gameListButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #E74C3C;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #C0392B;
      }
     </string>
    </property>
    <property name="text">
     <string>Create List</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputTextArea">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>661</width>
     <height>191</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>971</width>
     <height>871</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  QPushButton {
          border-radius: 20px;
      }</string>
   </property>
   <property name="pixmap">
    <pixmap>Images/background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>centralwidget</zorder>
  <zorder>statusbar</zorder>
  <zorder>outputTextArea</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
